ci/run_tests.sh

```diff
@@ -30,6 +30,11 @@ fi
 echo $PYTEST_CMD
 sh -c "$PYTEST_CMD"
 
+if [ "$COVERAGE" ]; then
```

Comment:
this is already above: https://github.com/pandas-dev/pandas/pull/40711/files#diff-fad2fa83c042cce646b2c1a0455e9b60c22dd5f105ed8859ce85dcf361c07fbdL12

Reply:
Yes, but this one is different. 

Now, maybe it can't hurt to add the `--cov-append` in all cases, though (so also for the first invocation)

Reply:
ok, but let's try to have a single invocation

Reply:
>  try to have a single invocation

There are two invocations, that's the point of also testing the ArrayManager